pkg/piecestore/rpc/server/store.go

```diff
@@ -54,6 +56,22 @@ func (s *Server) Store(reqStream pb.PieceStoreRoutes_StoreServer) (err error) {
 		return StoreError.New("failed to write piece meta data to database: %v", utils.CombineErrors(err, deleteErr))
 	}
 
+	if err = s.DB.AddBwUsageTbl(total, time.Now().Unix()); err != nil {
+		return StoreError.New("failed to write bandwidth info to database: %v", err)
+	}
+
+	bwSize, err := s.DB.GetBwUsageTbl(time.Now())
```

Comment:
we'll want to make sure to take this out with the comments

Reply:
Removed. Was added there during development.